ota-plus-web/app/com/advancedtelematic/controllers/UserProfileController.scala

```diff
@@ -54,6 +54,8 @@ class UserProfileController @Inject()(val conf: Configuration,
     }
 
     fut.recover {
+      case UnexpectedResponse(u) if (u.status == TOO_MANY_REQUESTS) =>
+        TooManyRequests(u.body)
```

Comment:
How will UI react on this request?

Reply:
LGTM. Should we wait for the UI change?

Reply:
I could also merge it now to see that the other error is fixed, i.e. prod and SIT show the same bug at least.